addons/io/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/Homekit.java

```diff
@@ -0,0 +1,10 @@
+package org.openhab.io.homekit;
```

Comment:
could you please run a `mvn license:format` on the project root in order to add the standard license headers on all files?